src/v2/guide/forms.md

```diff
@@ -88,7 +89,7 @@ new Vue({
 </script>
 {% endraw %}
 
-Multiple checkboxes, bound to the same Array:
+Checkboxes multiples, liées au même tableau (Array) :
```

Comment:
Moi ça me conviendrait. Il faudrait juste, si on traduit « checkboxes » par Cases à cocher traduire « Interpolation on textareas » par « Interpolation des textes multilignes ».

Par exemple « form inputs and textarea » parle selon mois des champs de formulaire de manière large et spécifiquement des champs textarea (pas de pluriel).